install/share/kdc.conf.template

```diff
@@ -21,3 +21,8 @@
   spake_preauth_indicator = hardened
   encrypted_challenge_indicator = hardened
  }
+
+[logging]
+ default = DEVICE:/dev/null
```

Comment:
This probably needs changed, unless I'm missing something?

Reply:
This disables the default logger. Logger configuration adds additional log locations and does not remove the default log files ``/var/log/krb5kdc.log``. Since krb5kdc is no longer allowed to write to ``/var/log`` I had to disable the default logger and only enable custom log locations.

Reply:
Per man page this shouldn't be necessary - default "Specifies how either daemon performs logging in the absence of relations specific to the daemon".

Reply:
In general I don't think I like that the log files move.  This is going to mess with admin expectations, if nothing else, and will not be the "usual" place for them.

Reply:
We can define 

```
[logging]
kdc = STDERR
admin_server = STDERR
```
then both krb5kdc and kadmind will automatically log into the journal.